### ISSUE TITLE
help not working

### ISSUE THREAD

**Issue by Mpako6ec** [finndev/PokeBuddy#828]
[P - 13:26:21] [Navigation] Walking from: 37.80673, -122.43149 to: 37.80693, -122.43129 with a distance of: 18 meters
[P - 13:26:27] [Navigation] Walking from: 37.80675, -122.43147 to: 37.80695, -122.43127 with a distance of: 18 meters
[P - 13:26:33] [Navigation] Walking from: 37.80677, -122.43145 to: 37.80697, -122.43125 with a distance of: 18 meters
[P - 13:26:39] [Navigation] Walking from: 37.80679, -122.43143 to: 37.80699, -122.43123 with a distance of: 18 meters
[P - 13:26:46] [Navigation] Walking from: 37.80681, -122.43141 to: 37.80701, -122.43121 with a distance of: 18 meters
[P - 13:26:52] [Navigation] Walking from: 37.80683, -122.43139 to: 37.80703, -122.43119 with a distance of: 18 meters
[P - 13:26:58] [Navigation] Walking from: 37.80685, -122.43137 to: 37.80705, -122.43117 with a distance of: 18 meters
[P - 13:27:04] [Navigation] Walking from: 37.80687, -122.43135 to: 37.80707, -122.43115 with a distance of: 18 meters
[P - 13:27:10] [Navigation] Walking from: 37.80689, -122.43132 to: 37.80709, -122.43112 with a distance of: 18 meters
[P - 13:27:17] [Navigation] Walking from: 37.80694, -122.43128 to: 37.80714, -122.43108 with a distance of: 18 meters
[P - 13:27:23] [Navigation] Walking from: 37.80696, -122.43125 to: 37.80716, -122.43105 with a distance of: 18 meters

I read 771 post but nothing helped

mb bot is down?

**Comment by Mpako6ec** [comment]
and i dont see pokestops

**Comment by DatFIXER** [comment]
seems you're soft banned bro

**Comment by DatFIXER** [comment]
#825

**Comment by Mpako6ec** [comment]
I tried did not help

**Comment by Mpako6ec** [comment]
Stop the bot
Set only "Farm Pokestops"
Let him run for 10 minutes
Works fine now


when he run 10min and more need reset ?

**Comment by DatFIXER** [comment]
you did all like i wrote right? wating times too? trying a fresh installation?

**Comment by DatFIXER** [comment]
no normally it goes.. try to shut down router/key/whatever connection you have, and try the "run method", lemme know (tagging @nickname)

**Comment by Mpako6ec** [comment]
yeah man i reinstall and install new client and do what you wrote

**Comment by r0nalxd** [comment]
If you don't see Pokestops you probably have an IP ban, see #707 for more info

**Comment by Mpako6ec** [comment]
1) reset internet adapter
2)use new client to only use pokestops 7km h
3)12min
4) dont see pokestops ) and [P - 13:26:21] [Navigation] Walking from: 37.80673, -122.43149 to: 37.80693, -122.43129 with a distance of: 18 meters
5) incebator off

**Comment by Mpako6ec** [comment]
@DatFIXER 
1) reset internet adapter
2)use new client to only use pokestops 7km h
3)12min
4) dont see pokestops ) and [P - 13:26:21] [Navigation] Walking from: 37.80673, -122.43149 to: 37.80693, -122.43129 with a distance of: 18 meters
5) incebator off

**Comment by DatFIXER** [comment]
both incubators off?
sorry for the many questions, trying to figure out :palm_tree:

**Comment by Mpako6ec** [comment]
yes incubator is off))

**Comment by DatFIXER** [comment]
Which folders have you deleted? All? and did a new extract from the original download file?

**Comment by Mpako6ec** [comment]
mb this not help me becouse in the work i have static ip!

**Comment by Mpako6ec** [comment]
yes i delete all files and extract new in folder

**Comment by Mpako6ec** [comment]
@DatFIXER Help man)

**Comment by T2HotZero** [comment]
I've got the same issue, even DLed that VPN thing... Doesn't work. After an hr or so i try and works for a min or two and then BOOM same issue. Every instruction followed DatFixer, no fix =/

**Comment by Mpako6ec** [comment]
@T2HotZero I understood now works only in your region where your ip is

**Comment by T2HotZero** [comment]
@Mpako6ec  so I live in the US.... In PA and boy in NY... Explain why I get banned pls?

**Comment by T2HotZero** [comment]
Bot
